helix-core/src/text_size/traits.rs

```diff
@@ -0,0 +1,25 @@
+use super::size::TextSize;
```

Comment:
I'm also confused here, I feel like this is also already a covered case w/ `RopeGraphemes` (I might be wrong though).